src/containers/my-quizzes/QuizzesContainer.tsx

```diff
@@ -109,18 +112,25 @@ const TestsContainer = () => {
     }
   ]
 
-  const table = (
-    <EnhancedTable
-      columns={columnsToShow}
-      data={{ items: response.items }}
-      emptyTableKey='myResourcesPage.quizzes.emptyQuizzes'
-      rowActions={rowActions}
-      sort={{
-        sort: { order: SortEnum.Desc, orderBy: 'updatedAt' },
-        onRequestSort: () => null
-      }}
-      sx={styles.table}
-    />
+  const tableWithPagination = (
+    <>
+      <EnhancedTable
+        columns={columnsToShow}
+        data={{ items: response.items }}
+        emptyTableKey='myResourcesPage.quizzes.emptyQuizzes'
+        rowActions={rowActions}
+        sort={{
+          sort: { order: SortEnum.Desc, orderBy: 'updatedAt' },
+          onRequestSort: () => null
+        }}
+        sx={styles.table}
+      />
+      <AppPagination
+        onChange={handleChangePage}
+        page={page}
+        pageCount={Math.ceil(response.count / itemsPerPage)}
+      />
+    </>
```

Comment:
To obtain new quizzes, you need to include a page in your request as the current method is ineffective.

Reply:
done